docs/modules/ROOT/partials/exoscale/delete-node.adoc

```diff
@@ -54,7 +56,7 @@ endif::[]
 +
 [source,bash,subs="attributes+"]
 ----
-for pv_name in $(echo -n {delete-pvs}); do
+for pv_name in $(echo -n ${delete-pvs}); do
```

Comment:
In (all) other templates the `$` is part of the attribute `:mon-replace-list: $NODE_TO_REMOVE`. Should we decide on one style?

Reply:
never mind then 🤦‍♀️